README.md

```diff
@@ -1,11 +1,18 @@
 # Tech Docs Template
 
-The Tech Docs Template is a [middleman template][mmt] that
+The Tech Docs Template is a [Middleman template][mmt] that
 you can use to build technical documentation using a GOV.UK style.
 
 This repo is the template used to generate new sites. The generated site uses HTML, CSS, JS and images from the [tech docs gem](https://github.com/alphagov/tech-docs-gem).
 
-👉 Find out more about the template and its features from the [Tech Docs Template documentation][tdt-docs].
+You’re welcome to use the template even if your service isn’t considered part of GOV.UK, but your site or service must not:
+
+- identify itself as being part of GOV.UK
```

Comment:
Do we want to point users in a more specific direction to make these changes e.g. the config.yml? Or is this sufficient

Reply:
There are more specific ways into the documentation about the config and the frontmatter further down, in the corresponding sections.

This is just a general introduction offering high level information about what all the different resources are.

I don't think there is a need for more detail in the introduction.

Reply:
agree